Dockerfile

```diff
@@ -1,50 +1,50 @@
-FROM ubuntu:trusty
+FROM isuper/java-oracle:jdk_8
```

Comment:
Docker does not support official Oracle JDK image: see section `Why is this only OpenJDK/OpenJRE?` in https://hub.docker.com/_/java/